website/static/css/custom.css

```diff
@@ -5,3 +5,47 @@
 table td:first-child > code {
   white-space: nowrap;
 }
+
```

Comment:
These CSS should be part of `main.css` as well so as to affect Docusaurus users.

Reply:
Fixed, I moved this to `lib/static/css/main.css`.